docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -170,3 +187,727 @@ Suggested workaround is to change the INET column to TEXT for the index creation
+
+**GitHub**: [Issue #1709](https://github.com/yugabyte/yugabyte-db/issues/1709)
+
+**Description**: If you have deferrable constraints on constraints other than foreign keys, for example, UNIQUE constraints which are currently not supported in the target YugabyteDB, it errors out in import schema phase as follows:
+
+```output
+ERROR: DEFERRABLE unique constraints are not supported yet
+```
+
+**Workaround**: Currrently, there is no workaround in target YugabyteDB, so remove it from the exported schema and modify the application for such constraints before pointing it on YugabyteDB.
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE public.users (
+    id int PRIMARY KEY,
+    email text
+);
+
+ALTER TABLE ONLY public.users
+    ADD CONSTRAINT users_email_key UNIQUE (email) DEFERRABLE;
+```
+
+---
+
+### Data ingestion on XML data type is not supported
+
+**GitHub**: [Issue #1043](https://github.com/yugabyte/yugabyte-db/issues/1043)
+
+**Description**: If you have XML datatype in source database, it errors out in import data to target YugabyteDB phase as data ingestion is not allowed on this data type:
+
+```output
+ ERROR: unsupported XML feature (SQLSTATE 0A000)
+```
+
+**Workaround**: To migrate the data, a workaround is to convert the type as text data type and get the data imported to target, but to read the data on target YugabyteDB you need to create some user defined functions similar to XML functions.
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE xml_example (
+      id integer,
+      data xml
+);
+```
+
+---
+
+### GiST index type is not supported
+
+**GitHub**: [Issue #1337](https://github.com/yugabyte/yugabyte-db/issues/1337)
+
+**Description**: If you have GiST indexes on the source database, it errors out in import schema phase in target with the following error:
+
+```output
+ ERROR: index method "gist" not supported yet (SQLSTATE XX000)
+
+```
+
+**Workaround**: Currently, there is no workaround, so remove it from the exported schema.
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE INDEX gist_idx ON public.ts_query_table USING gist (query);
+```
+
+---
+
+### Indexes on some complex data types are not supported
+
+**GitHub**: [Issue #9698](https://github.com/yugabyte/yugabyte-db/issues/9698)
+
+**Description**: If you have indexes on some complex types such as TSQUERY, TSVECTOR, JSON, UDTs, citext, and sp on, those will error out in import schema phase to target YugabyteDB with the following error:
+
+```output
+ ERROR:  INDEX on column of type '<TYPE_NAME>' not yet supported
+```
+
+**Workaround**: Currently, there is no workaround, but you can cast these data types to supported types, which may require adjustments on the application side to query the column using the index. Ensure you address these changes before modifying the schema.
```

Comment:
Currently, there is no workaround, but you can cast these data types in the index definition to supported types, which may require adjustments on the application side while querying the column using the index. Ensure you address these changes before modifying the schema